### PR TITLE
cw1-subkeys: Point helper to smart contract version v0.2.1

### PR DESCRIPTION
**Pull request by orkunkl**
First, merge https://github.com/CosmWasm/cosmwasm-plus/pull/83 if no objections or blockers
Then release v0.2.1
Then merge this.

### CODE REVIEW

File: contracts/cw1-subkeys/helpers.ts [hunk 1]
```diff
@@ -346,10 +346,10 @@ const CW1 = (client: SigningCosmWasmClient): CW1Contract => {
   
   const upload = async (): Promise<number> => {
     const meta = {
-      source: "https://github.com/CosmWasm/cosmwasm-plus/tree/v0.1.1/contracts/cw1-subkeys",
+      source: "https://github.com/CosmWasm/cosmwasm-plus/tree/v0.2.1/contracts/cw1-subkeys",
       builder: "cosmwasm/rust-optimizer:0.10.1"
```
Comment (orkunkl) [inline contracts/cw1-subkeys/helpers.ts:350]:
```suggestion
      builder: " cosmwasm/workspace-optimizer:0.10.3"
```
Should I apply this change?

Comment (orkunkl) [reply contracts/cw1-subkeys/helpers.ts:350]: I suppose we change the meta in every contract after we apply this change.

**Review by ethanfrey — APPROVED** [review]
Please update to `cosmwasm/workspace-optimizer:0.10.2` in both helper.ts files, then good to merge

**Comment by ethanfrey** [comment]
Just merged #83 Now I see why you changed this.

**Pull request merged**